ops/dev7/persistent/_config.tf

```diff
@@ -3,19 +3,19 @@ terraform {
     resource_group_name  = "prime-simple-report-test"
     storage_account_name = "usdssimplereportglobal"
     container_name       = "sr-tfstate"
-    key                  = "dev7/persistent-terraform.tfstate"
+    key                  = "dev/persistent-terraform.tfstate"
```

Comment:
Does this need to be `dev7` instead of `dev`?

Reply:
Thanks for catching this!

Reply:
fixed